images/installer/Dockerfile.upi.ci

```diff
@@ -56,7 +56,7 @@ ENV IGNITION_PROVIDER_VERSION=v2.1.0
 RUN curl -L -O https://github.com/community-terraform-providers/terraform-provider-ignition/releases/download/${IGNITION_PROVIDER_VERSION}/terraform-provider-ignition-${IGNITION_PROVIDER_VERSION}-linux-amd64.tar.gz && \
     tar xzf terraform-provider-ignition-${IGNITION_PROVIDER_VERSION}-linux-amd64.tar.gz && \
     mv terraform-provider-ignition-${IGNITION_PROVIDER_VERSION}-linux-amd64/terraform-provider-ignition /bin/terraform-provider-ignition
-RUN curl -L -o - "https://github.com/vmware/govmomi/releases/latest/download/govc_$(uname -s)_$(uname -m).tar.gz" | tar -C /bin -xvzf - govc
+RUN curl -L -o - "https://github.com/vmware/govmomi/releases/latest/download/govc_$(uname -s)_$(uname -m).tar.gz" | tar -C /bin -xvf - govc
```

Comment:
Was this causing any issues? I grabbed their latest and could still unzip as if it were compressed with gzip using `-xzvf`. The filename also appears to still use the tar.gz values.

Reply:
I had to ask because the job doesn't appear to load on my computer the link is fine but it just keeps spinning.

Reply:
It was saying this yesterday in the e2e-metal-upi. Probably some kind of build error or link failure kind of thing but can't verify now with the test links all spinning
```
gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error exit delayed from previous errors
```

Reply:
Interesting. Technically either command worked for me when I pulled it.